src/lib.rs

```diff
@@ -1365,4 +1365,18 @@ mod tests {
             assert_eq!(output, "oof");
         }
     }
+
+    mod invocation_syntax {
+        use super::*;
+
+        #[test]
+        fn trailing_comma_is_accepted_after_normal_argument() {
+            cmd_unit!("echo", "foo",);
+        }
+
+        #[test]
+        fn trailing_comma_is_accepted_after_split_argument() {
+            cmd_unit!("echo", %"foo",);
+        }
```

Comment:
Maybe it'd be good to also have one test for `cmd!` and one for `cmd_result!`. So that all unhidden macros are covered.